runtime/doc/editing.txt

```diff
@@ -1615,7 +1615,7 @@ There are three different types of searching:
    directory or in the directory of the current file (if the relative path
    starts with './' and 'd' is not included in 'cpoptions').
 
-   If Vim's current path is /u/user_x/work/release and you do >
+   If Vim's cu rrent path is /u/user_x/work/release and you do >
```

Comment:
?

Reply:
Wow. I have no idea how that happened or how I missed that when checking the diff. Fixed in 24b35fe9d4b397e862178f8b8725b4631d41533d